cmd/ethkey/utils.go

```diff
@@ -140,7 +140,7 @@ func ReadYesOrNoAnswer(message string) (bool, error) {
 	if input == "" || input == "y" || input == "Y" || input == "yes" || input == "Yes" || input == "YES" {
 		result = true
 	} else if input != "n" && input != "N" && input != "no" && input != "No" && input != "NO" {
-		return false, errors.New(fmt.Sprintf("You entered a wrong answer: %s. Aborting execution", input))
+		return false, fmt.Errorf("You entered a wrong answer: %s. Aborting execution", input)
```

Comment:
Just an FYI since you're just updating existing code - Errors should be concise and wrap nicely. A potentially better replacement here would be: "invalid input: %s". Let the calling function handle saying things like aborting execution.